test/components/table/list-view.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+import test from 'ava';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import {shallow, configure} from 'enzyme';
+import {listGranules} from '../../../app/scripts/actions';
+import List from '../../../app/scripts/components/table/list-view';
+import Timer from '../../../app/scripts/components/app/timer.js';
+import {
+  errorTableHeader,
+  errorTableRow, errorTableSortProps
+} from '../../../app/scripts/utils/table-config/granules';
+
+configure({ adapter: new Adapter() });
+
+test('list-view should properly initialize timer config prop', async (t) => {
+  const dispatch = () => Promise.resolve();
+  const query = { q: '_exists_:error AND status:failed' };
```

Comment:
Nice! So this proves what we care about - that when we pass in a query (really any query), it should be properly initialized by the component without any lifecycle methods

Reply:
Precisely. I just wish we had found the `disableLifecycleMethods` option for the `shallow` method sooner :-)